pending/smplayer.xml

```diff
@@ -293,43 +303,57 @@
     <action command="ini" search="file" path="$$profile$$\playlist.ini" section="playlist_contents" parameter="items\27\item_26_params"/>
     <action command="ini" search="file" path="$$profile$$\playlist.ini" section="playlist_contents" parameter="items\27\item_26_video_url"/>
     <action command="ini" search="file" path="$$profile$$\playlist.ini" section="playlist_contents" parameter="items\27\item_26_icon_url"/>
+  <!-- That we don't need so much translation, we combine all in "History"! Tobias.
   </option>
   <option id="screenshot_folder">
     <label>Screenshots path</label>
     <description>Reset screenshots path</description>
+  -->
     <action command="ini" search="file" path="$$profile$$\smplayer.ini" section="%General" parameter="screenshot_folder"/>
-  </option>
-  <option id="screenshots">
-    <label>Screenshots</label>
-    <description>Remove screenshots image files from default location</description>
-    <action command="delete" search="glob" path="$$profile$$\screenshots\*"/>
-    <action command="delete" search="glob" path="%Pictures%\smplayer*_screenshots\*"/>
+  <!-- That we don't need so much translation, we combine all in "History"! Tobias.
   </option>
   <option id="history_files">
     <label>Recent Files</label>
     <description>Clear recent played files</description>
+  -->
     <action command="ini" search="file" path="$$profile$$\smplayer.ini" section="history" parameter="recents"/>
+  <!-- That we don't need so much translation, we combine all in "History"! Tobias.
   </option>
   <option id="history_urls">
     <label>Recent URLs</label>
```

Comment:
You could use the two MRU strings that are already translated, if you want

> Most recently used
> Delete the most recently used list

Reply:
As this option includes e.g. now `Remove file settings (last position, subtitles size, fonts)`, I decided to use `History` & `Delete the usage history` in my latest version... but I haven't pushed it by now... :-/
IMHO this strings fits most and are already translated... Do they work for you, too?